packages/keyring-snap-bridge/package.json

```
@@ -32,26 +32,26 @@
     "@metamask/snaps-utils": "^7.8.1",
     "@metamask/superstruct": "^3.1.0",
     "@metamask/utils": "^9.2.1",
-    "@types/uuid": "^9.0.1",
-    "uuid": "^9.0.0"
+    "@types/uuid": "^9.0.8",
+    "uuid": "^9.0.1"
   },
   "devDependencies": {
     "@lavamoat/allow-scripts": "^3.2.1",
     "@metamask/auto-changelog": "^3.4.4",
-    "@types/jest": "^28.1.6",
-    "@types/node": "^17.0.23",
+    "@types/jest": "^29.5.12",
+    "@types/node": "^20.12.12",
     "deepmerge": "^4.2.2",
-    "depcheck": "^1.4.3",
-    "jest": "^28.1.3",
-    "jest-it-up": "^2.0.2",
-    "rimraf": "^3.0.2",
-    "ts-jest": "^28.0.7",
-    "ts-node": "^10.7.0",
-    "typedoc": "^0.23.15",
+    "depcheck": "^1.4.7",
+    "jest": "^29.5.0",
+    "jest-it-up": "^3.1.0",
+    "rimraf": "^5.0.7",
+    "ts-jest": "^29.0.5",
+    "ts-node": "^10.9.2",
+    "typedoc": "^0.25.13",
     "typescript": "~4.8.4"
   },
   "peerDependencies": {
-    "@metamask/keyring-api": "^8.1.1"
+    "@metamask/keyring-api": "8.1.2"
```

Should this be using caret notation?

```suggestion
    "@metamask/keyring-api": "^8.1.2"
```